_meta/beat.reference.yml

```diff
@@ -21,8 +21,11 @@ apm-server:
   #ssl.certificate : "path/to/cert"
   #ssl.key : "path/to/private_key"
 
+  #frontend.enabled: false
+  #frontend.rate_limit: 10
```

Comment:
You used to have a little note here explaining that it's "per second" and that zero means disabled. Would be great to get that back.

Reply:
per seconds: yes, absolutely
zero means disabled: i wanted to use that as configuration for enable/disable frontend, but it was rejected. it seams silly now to say you can use `frontend.enabled = false` _and/or_  `frontend.rate_limit = 0` to disable frontend.

Reply:
what do you suggest is a good way to disable rate limiting?

Reply:
what it was considered is `rate_limit = 0` to disabled frontend. 
i don't think there was a note or option to have frontend enabled and rate limit disabled. you mean that? why would someone _want_ that? 
still, if you want it, set it to a really high number like `10000000`, but i think is silly to document that.

Reply:
yeah, that's what I mean. I think we'll have some people telling they want to disable the rate limit completely. We'll need to tell them there's no way to disable, but they can set it very high - which also seems a bit silly.
I'm OK to merge now and we can see if this ever becomes a problem.